tests/lib/Encoder/Field/PageBuilderFieldEncoderTest.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\AutomatedTranslation\Encoder\Field;
+
+use eZ\Publish\API\Repository\Values\Content\Field;
+use eZ\Publish\Core\Repository\ProxyFactory\ProxyGenerator;
+use EzSystems\EzPlatformAutomatedTranslation\Encoder\BlockAttribute\BlockAttributeEncoderManager;
+use EzSystems\EzPlatformAutomatedTranslation\Encoder\Field\PageBuilderFieldEncoder;
+use EzSystems\EzPlatformPageFieldType\FieldType\LandingPage\Model\Attribute;
+use EzSystems\EzPlatformPageFieldType\FieldType\LandingPage\Model\BlockValue;
+use EzSystems\EzPlatformPageFieldType\FieldType\LandingPage\Model\Page;
+use EzSystems\EzPlatformPageFieldType\FieldType\LandingPage\Model\Zone;
+use EzSystems\EzPlatformPageFieldType\FieldType\LandingPage\Value;
+use EzSystems\EzPlatformPageFieldType\FieldType\Page\Block\Definition\BlockAttributeDefinition;
+use EzSystems\EzPlatformPageFieldType\FieldType\Page\Block\Definition\BlockDefinition;
+use EzSystems\EzPlatformPageFieldType\FieldType\Page\Block\Definition\BlockDefinitionFactory;
+use PHPUnit\Framework\TestCase;
+use ProxyManager\Proxy\LazyLoadingInterface;
+
+class PageBuilderFieldEncoderTest extends TestCase
+{
+    public const ATTRIBUTE_VALUE = 'ibexa';
+
+    public function testEncode(): void
+    {
+        $blockAttributeEncoderManagerMock = $this->getMockBuilder(BlockAttributeEncoderManager::class)
+            ->disableOriginalConstructor()
+            ->getMock();
```

Comment:
If you want to write fewer lines you can do:

```suggestion
        $blockAttributeEncoderManagerMock = $this->createMock(BlockAttributeEncoderManager::class);
```